ci/infra/aws/security-groups.tf

```diff
@@ -208,6 +243,14 @@ resource "aws_security_group" "elb" {
     "Class", "SecurityGroup"))}"
 
   # HTTP access from anywhere
+  ingress {
+    from_port   = 80
+    to_port     = 80
```

Comment:
Is this in preparation for the future ingress controller support? [2]

Reply:
yes